R/estimate_secondary.R

```diff
@@ -238,15 +238,17 @@ secondary_opts <- function(type = "incidence", ...) {
       historic = 1,
       primary_hist_additive = 1,
       current = 0,
-      primary_current_additive = 0
+      primary_current_additive = 0,
+      estimate_baseline = 0
     )
   } else if (type %in% "prevalence") {
     data <- list(
       cumulative = 1,
       historic = 1,
       primary_hist_additive = 0,
       current = 1,
-      primary_current_additive = 1
+      primary_current_additive = 1,
+      estimate_baseline = 0
```

Comment:
better to add this to the list outside of the if-else step to avoid duplication.